### PR TITLE
Add corelight/zeek-community-id package

### LINKED ISSUE #36: Community ID support
We'd like to enhance the Zeek artifact that's bundled with Brim to add the Community ID field to `conn` records. This will allow for joining with other data sources, such as Suricata (https://github.com/brimsec/zq/issues/1211).

### PR DESCRIPTION
**Pull request by nwt**
The corelight/zeek-community-id package includes a Zeek plugin.
Building and loading plugins doesn't work on Windows, so this change
consists almost entirely of jiggery-pokery to address that, including a
modified cmake submodule at https://github.com/brimsec/cmake.

Should close #36 and #37.

### CODE REVIEW

File: zeek-config.in [hunk 1]
```diff
@@ -5,13 +5,13 @@ build_type=@CMAKE_BUILD_TYPE_LOWER@
 prefix=@CMAKE_INSTALL_PREFIX@
 script_dir=@ZEEK_SCRIPT_INSTALL_PATH@
 site_dir=@ZEEK_SCRIPT_INSTALL_PATH@/site
-plugin_dir=@BRO_PLUGIN_INSTALL_PATH@
+plugin_dir=@bro_plugin_install_path@
```
Comment (alfred-landrum) [inline zeek-config.in:8]: Are the upper -> lower case changes here, and in zeek-path-dev.in, intentional?

File: CMakeLists.txt [hunk 2]
```diff
@@ -78,20 +78,35 @@ get_filename_component(ZEEK_SCRIPT_INSTALL_PATH ${ZEEK_SCRIPT_INSTALL_PATH}
 
 set(BRO_PLUGIN_INSTALL_PATH ${ZEEK_ROOT_DIR}/lib/zeek/plugins CACHE STRING "Installation path for plugins" FORCE)
 
+set(bro_plugin_install_path "${BRO_PLUGIN_INSTALL_PATH}")
```
Comment (alfred-landrum) [inline CMakeLists.txt:81]: I just noticed this after making my comment on zeek-config.in. So these are paths that have been cleared of any windows volume identifiers?

Comment (nwt) [reply CMakeLists.txt:81]: Yes, though the volume identifiers have been MSYS-ified (by converting c:/ to /c/) rather than replaced.

**Comment by philrz** [comment]
I think we're in good shape here from a functional perspective. I tested things out by manually downloading the macOS Zeek artifact from the most recent Actions run on this branch (https://github.com/brimsec/zeek/suites/1299072896/artifacts/20371620). In my local dev Brim environment, I removed the regular `zdeps/zeek/` directory replaced it with the one from this artifact. Manually importing pcaps into Brim, I can see the Community ID field is populated as we'd expect:

image

I also made sure the other Zeek packages are still working as expected.

JA3:

image

HASSH:

image

geoip-conn:

image

I also did an `npm run itest` to make sure that none of the Brim CI that deals with Zeek logs generated from pcaps were disturbed by the addition of the new Community ID field, and as I'd hoped, everything passed without complaint.

```
Test Suites: 9 passed, 9 total
Tests:       104 passed, 104 total
Snapshots:   54 passed, 54 total
Time:        222.266s, estimated 223s
Ran all test suites.
```

In conclusion, if you guys are ok with this at a code level, I'm game to see it merged so we can start delivering Community ID as a new field even in advance of the rest of Suricata support being ready. We've already had validation from our user community that it's helpful just to have the value from it even if they're manually cut & pasting the value out of Brim and into some other pane-of-glass where data is stored that has Community ID as a field.

**Review by alfred-landrum — APPROVED** [review]

**Review by philrz — APPROVED** [review]
